src/Parser.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace CSVParser;
+
+class Parser {
+
+    var $path;
+    var $header_array = array();
+    var $delimiter;
+
+    function __construct ($path, $header_array = array(), $delimiter = ",") {
+
+        if(file_exists($path)){ //verify path
+            $this -> $path = $path; //fill path
+
+            if($delimiter == "," || $delimiter == "|" || $delimiter == "\t") { //verify delimiter
+                $this -> $delimiter = $delimiter; //fill delimiter

```

Comment:
drop the spaces in front of and after the `->`please. It's stylistic but no one does it this way.